eng/common/scripts/Helpers/DevOps-WorkItem-Helpers.ps1

```diff
@@ -802,4 +802,21 @@ function UpdatePackageVersions($pkgWorkItem, $plannedVersions, $shippedVersions)
   $response = Invoke-RestMethod -Method PATCH `
     -Uri "https://dev.azure.com/azure-sdk/_apis/wit/workitems/${id}?api-version=6.0" `
     -Headers $headers -Body $body -ContentType "application/json-patch+json"
+}
+
+function Get-LanguageName($lang)
+{
+    $pkgLang = $languageNameMapping[$lang]
+    if (!$pkgLang) { 
+      $pkgLang = $lang
+    }
+    return $pkgLang
+}
+
+$languageNameMapping = @{
+    cpp = "C++"
```

Comment:
Can you go ahead and add the other languages here as well as we will need them eventually